cli/worker.rs

```diff
@@ -31,11 +34,251 @@ impl Worker {
     let isolate = Arc::new(Mutex::new(deno::Isolate::new(startup_data, false)));
     {
       let mut i = isolate.lock().unwrap();
-
       let state_ = state.clone();
-      i.set_dispatch(move |op_id, control_buf, zero_copy_buf| {
-        state_.dispatch(op_id, control_buf, zero_copy_buf)
-      });
+
+      i.register_op("read", state_.cli_op(minimal_op(io::op_read)));
+      i.register_op("write", state_.cli_op(minimal_op(io::op_write)));
+
+      i.register_op(
+        "exit",
+        state_.cli_op(json_op(state_.stateful_op(os::op_exit))),
+      );
+      i.register_op(
+        "is_tty",
+        state_.cli_op(json_op(state_.stateful_op(os::op_is_tty))),
+      );
+      i.register_op(
+        "env",
+        state_.cli_op(json_op(state_.stateful_op(os::op_env))),
+      );
+      i.register_op(
+        "exec_path",
+        state_.cli_op(json_op(state_.stateful_op(os::op_exec_path))),
+      );
+      i.register_op(
+        "utime",
+        state_.cli_op(json_op(state_.stateful_op(fs::op_utime))),
+      );
+      i.register_op(
+        "set_env",
+        state_.cli_op(json_op(state_.stateful_op(os::op_set_env))),
+      );
```

Comment:
This is ripe for abstraction - we need to do it anyway, because there we still need to provide different ops for different workers (see #1858)